src/index.js

```diff
@@ -95,11 +95,11 @@ interface Platform {
   /**
   * The runtime's location API.
   */
-  location: Object;
+  location: typeof window.location;
   /**
   * The runtime's history API.
   */
-  history: Object;
+  history: typeof window.history;
```

Comment:
Is this correct for history and location? I get that `Element` isn't an instance of `Element` but is the type itself but history and location are the instances of history and location.